setup.py

```diff
@@ -384,7 +384,7 @@ def gen_pyx(odir):
             'Topic :: Software Development',
           ],
         setup_requires = ['numpy>=1.14', 'cython', 'jinja2'],
-        install_requires = ['numpy>=1.14', 'tbb', 'daal', 'dpcpp_cpp_rt'] + ['impi_rt'] if not "osx" in sys.platform else [],
+        install_requires = ['numpy>=1.14', 'tbb', 'daal', 'dpcpp_cpp_rt'] + ['impi_rt'] if "osx" not in sys.platform else [],
```

Comment:
```suggestion
        install_requires = ['numpy>=1.14', 'tbb', 'daal', 'dpcpp_cpp_rt'] + ['impi_rt'] if not IS_MAC else [],
```

It seems to me more and more that getting rid of the RT dependency of the MPI looks like a nice and correct solution :)